src/pb_stub.cc

```diff
@@ -712,13 +715,15 @@ main(int argc, char** argv)
 
   stub->Initialize(model_version, argv[6] /* triton install path */);
   bool non_graceful_exit = false;
+  bool &health = stub->Health();
 
   bool background_thread_running = true;
   std::thread background_thread(
-      [&parent_pid, &background_thread_running, &stub, &non_graceful_exit] {
+      [&parent_pid, &background_thread_running, &stub, &non_graceful_exit, &health] {
         while (background_thread_running) {
-          // Every two seconds check if the parent process is alive.
+          // Every 10ms check if the parent process is alive.
```

Comment:
Does it really need to reset the health every 10ms? That seems unnecessarily frequent.

Reply:
Yes, that is not really required. I'll increase it to 300ms so that it has 3 chances to set this variable when the parent process is polling this variable (parent process polls this variable every 1s).